phpBB/config/production/container/services.yml

```diff
@@ -1,2 +1,8 @@
 imports:
     - { resource: ../../default/container/services.yml }
+
+services:
```

Comment:
I need the language service in the installer, and probably I cannot have user there (I'm not sure but probably something has dependency of the from the user dependencies that I cannot have there)